src/models/cloudProviderIdentifier.ts

```diff
@@ -13,13 +13,26 @@ export enum CloudAdapterIdentifier {
 }
 
 export enum CloudProviderIdentifier {
-    GENEZIO = "genezio",
+    CAPYBARA_LINUX = "genezioCloud",
+    CAPYBARA = "genezioCloudUnikernel",
+    CLUSTER = "genezioCluster",
+    GENEZIO = "genezioAws",
```

Comment:
I would rename these variables properly.

```suggestion
    GENEZIO_CLOUD = "genezioCloud",
    GENEZIO_UNIKERNEL = "genezioCloudUnikernel",
    GENEZIO_CLUSTER = "genezioCluster",
    GENEZIO_AWS = "genezioAws",
```

Reply:
Done, thanks!